app/views/transactionDetails.html

```diff
@@ -154,8 +155,8 @@ <h3 class="panel-title">Response</h3>
                 </div>
                 <div class="panel-body">
 
-                  <span class="label label-danger" ng-if="channel.type !== 'http'">{{channel.type | uppercase}}</span>
-                  <h4 ng-if="channel.type === 'http'">Status code
+                  <span class="label label-info" ng-if="channel.type !== 'http' && channel.type !== 'polling'">{{channel.type | uppercase}}</span>
```

Comment:
Not sure if want to display polling in the response panel too?  It's already displayed in the request panel.